engine/src/flutter/lib/ui/painting.dart

```diff
@@ -3,6 +3,11 @@
 // found in the LICENSE file.
 part of dart.ui;
 
+// TODO(loic-sharma): Migrate fragment shader samples off deprecated
+// Color APIs.
+// See: https://github.com/flutter/flutter/issues/179146
+// ignore_for_file: deprecated_member_use
```

Comment:
FYI this turns off deprecation warnings for the whole file. I plan to fast-follow with a fix today, after we reopen the tree.

There's some non-trivial fragment shader samples below that uses deprecated Color APIs. This is complex enough that I'd like this to be reviewed separately.

Reply:
For whatever reason, I can't get the deprecation lint suppression to work on this file. I ended up migrating the fragment shader examples as part of this PR. @gaaclarke reviewed the changes offline.